examples/experimental/onnx/semantic_segmentation/README.md

```diff
@@ -0,0 +1,62 @@
+# Semantic Segmentation sample
+
+This sample shows an example of quantization of segmentation models. The used datasets are CamVid, Mapillary.
```

Comment:
Please replace "This sample shows an example" rhyme with something, like "This example demonstrates quantization of ...". In this case, it is better to change the title "Semantic Segmentation Example"

Reply:
I would even specify what kind of models, like  "This example demonstrates quantization of UNet and ICNet segmentation models on CamVid and Mapillary datasets."